dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/registry/ServerNodeManager.java

```diff
@@ -241,8 +235,7 @@ private void updateMasterNodes() {
         try {
             registryClient.getLock(nodeLock);
             Collection<String> currentNodes = registryClient.getMasterNodesDirectly();
-            List<Server> masterNodes = registryClient.getServerList(NodeType.MASTER);
-            syncMasterNodes(currentNodes, masterNodes);
+            syncMasterNodes(currentNodes, registryClient.getServerList(NodeType.MASTER));
```

Comment:
There is no need to do this code change.

Reply:
> There is no need to do this code change.

Because `masterNodes` local variable should not shadow class fields. Override or shadow a variable declared in the outer scope can strongly impact the readability,and it will lead mantainers to introduce bugs because they think they are using one variable but are really using another. WDYT.

Reply:
Make sense.